src/main/java/pl/edu/mimuw/Main.java

```diff
@@ -1,7 +1,23 @@
 package pl.edu.mimuw;
 
+import pl.edu.mimuw.chess.*;
+
 public class Main {
 
-  public static void main(String[] args) {
+  public static void main(String[] args) throws InterruptedException {
+    ChessGame game = new ChessGame();
+    White white = game.getWhite();
+    Black black = game.getBlack();
+
+    while (game.getMoveCount() < 100 && game.isNotFinished())
```

Comment:
starajmy się wyciągać stałe, nawet jak raz się pojawiają (tu jeszcze to zatem tak nie boli)